readme.md

```diff
@@ -0,0 +1,153 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>README</b></h3>
+</div>
+
+<br>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+
+Hello Microverse is the first miniproject from module 1. This project serves to familiarize students with Git, Github,gitflow and linters.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**

```

Comment:
- [ ] Please customize this section and the others as well 🥰. Documentation is used as a guide for our users so it should be meaningful 🥸
For example: in the **key features** section you can mention the **use of Linters** because that is what is done for this project.